### DISCUSSION TITLE
Symmetry Condition for Plane not Aligned with Cartesian Planes

### DISCUSSION THREAD

**Discussion by elementx54** [idaholab/moose#18135] [Q&A Modules: Solid mechanics] [answered]
I've got a cylindrical mesh that has a symmetry wedge from 0 to 72 degrees. I'd like to use symmetric boundary conditions to avoid having to simulate the entire cylinder. I understand how to set displacements to zero for symmetric conditions when a symmetry plane is orthogonal to the displacement axis in Cartesian coordinates

So my question, is there a boundary condition for tensor mechanics that will allow me to set symmetry displacement conditions on my 72 degree surface whose symmetry plane is not orthogonal to any Cartesian displacement?

My geometry is 3D with features that do not allow it to be 2D, 1D, 2D RZ, or 1D RZ.

Thanks!

**Comment by hugary1995** [comment, marked as answer]
Take a look at inclined no disp BCs [https://mooseframework.inl.gov/syntax/BCs/InclinedNoDisplacementBC/index.html](https://mooseframework.inl.gov/syntax/BCs/InclinedNoDisplacementBC/index.html#!)

**Reply by elementx54** [reply]
I tried that with a penalty of 1e9 and my surface did not stay in the plane. Do I just need a higher penalty? Can you put in too high of a penalty?

**Reply by hugary1995** [reply]
Most likely yes. But this could also happen if your penalty is too high. I would start with a penalty that is 1x~100x the young's modulus. Automatic scaling also helps to some extent.

Well, this is the common problem with penalty methods. I remember someone was developing a Nitsche's version, but I don't know the current status of that.
